server.js

```diff
@@ -45,6 +45,13 @@ app.prepare().then(() => {
             : pathname)
       })
       res.end()
+    } else if (req.headers.host === 'error.dvc.org') {
```

Comment:
p.s. @shcheklein please setup the error.dvc.org DNS record when you can.

Reply:
@jorgeorpinel @pared done!